collections/hub/sdk-bulk/bulk-happy-path.json

```diff
@@ -53,51 +54,52 @@
                 "to": {
                   "partyIdInfo": {
                     "partyIdType": "MSISDN",
-                    "partyIdentifier": "56781234001"
+                    "partyIdentifier": "{$inputs.SIM2_MSISDN_1}"
                   }
                 },
                 "amountType": "SEND",
-                "currency": "USD",
-                "amount": "100"
+                "currency": "{$inputs.SIM2_CURRENCY}",
+                "amount": "456.78"
               },
               {
                 "homeTransactionId": "abc2",
                 "to": {
                   "partyIdInfo": {
                     "partyIdType": "MSISDN",
-                    "partyIdentifier": "56781234002"
+                    "partyIdentifier": "{$inputs.SIM2_MSISDN_2}"
                   }
                 },
                 "amountType": "SEND",
-                "currency": "USD",
-                "amount": "100"
+                "currency": "{$inputs.SIM2_CURRENCY}",
+                "amount": "678.91"
               },
               {
                 "homeTransactionId": "abc3",
                 "to": {
                   "partyIdInfo": {
                     "partyIdType": "MSISDN",
-                    "partyIdentifier": "56781235001"
+                    "partyIdentifier": "{$inputs.SIM3_MSISDN_1}"
                   }
                 },
                 "amountType": "SEND",
-                "currency": "USD",
-                "amount": "100"
+                "currency": "{$inputs.SIM3_CURRENCY}",
+                "amount": "678.91"
               },
               {
                 "homeTransactionId": "abc4",
                 "to": {
                   "partyIdInfo": {
                     "partyIdType": "MSISDN",
-                    "partyIdentifier": "56781235002"
+                    "partyIdentifier": "{$inputs.SIM3_MSISDN_2}"
                   }
                 },
                 "amountType": "SEND",
-                "currency": "USD",
-                "amount": "100"
+                "currency": "{$inputs.SIM3_CURRENCY}",
+                "amount": "678.91"
               }
             ]
           },
+          "url": "{$inputs.SIM1_SDK_BACKEND_ENDPOINT}",
```

Comment:
The "Override with Custom URL" should not be checked in all the test cases. i.e, you need to remove this parameter

Reply:
Please remove this in all the places

Reply:
@vijayg10 ..if this is removed, what will be the default value of the URL?

Reply:
The default value is already configured in the TTK settings. So we don't need the URL in all the test cases.

Reply:
What is the default value? @vijayg10

Reply:
Also is there any harm in leaving this value? Because my plan is to use these collections both in local and in all the environments without any changes to the collections. In local env, we need this value. @vijayg10